lib/ansible/modules/cloud/amazon/ec2_customer_gateway.py

```diff
@@ -255,6 +274,9 @@ def main():
                     results['gateway']['CustomerGateway']['CustomerGatewayId'],
                     module.params['name'],
                 )
+                created = gw_mgr.describe_gateways(module.params['ip_address'])
+                tag_array = created['CustomerGateways'][0]['Tags']
+                results['gateway']['CustomerGateway']['Tags'] = tag_array
```

Comment:
Really need to run `boto3_tag_list_to_ansible_dict` on `tag_array`, but best to do that *after* `camel_dict_to_snake_dict` (otherwise tag keys get converted to lower case which can be unexpected)